### DISCUSSION TITLE
A simple project made with customtkinter

### DISCUSSION THREAD

**Discussion by receyuki** [TomSchimansky/CustomTkinter#1798] [Showcase]
Repo: https://github.com/receyuki/stable-diffusion-prompt-reader

I tried my best to make it looks great.
<img width="1287" alt="sdpr" src="https://github.com/TomSchimansky/CustomTkinter/assets/28808141/000fe582-bdb6-443c-b364-642264116838">

**Comment by avalon60** [comment]
Hi @receyuki, what does it do?

**Reply by receyuki** [reply]
> A simple standalone viewer for reading prompt from Stable Diffusion generated image outside the webui.

A little tool for stable diffusion content creators

**Reply by avalon60** [reply]
Never heard of Stable Diffusion before; just Googled it. Impressive!

**Reply by receyuki** [reply]
I was also impressed when I realised that images I previously thought were photos of real people were actually generated by SD.
